flink-connectors/flink-connector-kafka-0.10/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaTestEnvironmentImpl.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.flink.streaming.util.serialization.KeyedSerializationSchema;
 import org.apache.flink.util.NetUtils;
 
+import com.google.common.collect.ImmutableList;
```

Comment:
In Flink we usually try to avoid Guava usages. Would it be easy to switch to `Collections.unmodifiableList`?